common/trinary/trit_long.c

```diff
@@ -29,22 +29,37 @@ static double const pow27LUT[] = {1,
                                   150094635296999136,
                                   4052555153018976256};
 
-int64_t trits_to_long(trit_t const *const trit, size_t const i) {
+int64_t trits_to_long(trit_t const *const trits, size_t const num_trits) {
+  if (num_trits == 0) {
+    return 0;
+  }
   int64_t accum = 0;
-  size_t end = i;
+  size_t end = num_trits;
   while (end-- > 0) {
-    accum = accum * RADIX + trit[end];
+    accum = accum * RADIX + trits[end];
   }
   return accum;
 }
 
 size_t min_trits(int64_t const value) {
+  printf("Value: %lld\n", value);
```

Comment:
remove these debug statements please

Reply:
Sure, I'm debugging right now because it's failing with -c opt...